### ISSUE TITLE
Can't install on Craft 5.0.2. 

### ISSUE THREAD

**Issue by JMKelley** [verbb/consume#12]
### Describe the bug

```
Composer was unable to install the updates.

Error: The command "'/usr/bin/php' '/var/www/html/storage/runtime/composer.phar' 'update' 'verbb/consume' '--with-all-dependencies' '--working-dir' '/var/www/html' '--no-scripts' '--no-ansi' '--no-interaction'" failed.

Exit Code: 2(Misuse of shell builtins)

Working directory: /var/www/html/web

Output:
================

Error Output:
================
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

Problem 1
- Conclusion: don't install craftcms/cms 5.0.2 (conflict analysis result)
- verbb/consume 2.0.0-beta.1 requires symfony/property-access ^6.0 -> satisfiable by symfony/property-access[v6.0.0-BETA1, ..., 6.4.x-dev].
- You can only install one version of a package, so only one of these can be installed: symfony/property-access[v6.0.0-BETA1, ..., 6.4.x-dev, v7.0.0-BETA1, ..., 7.1.x-dev].
- craftcms/cms 5.0.2 requires symfony/property-access ^7.0 -> satisfiable by symfony/property-access[v7.0.0-BETA1, ..., 7.1.x-dev].
- Conclusion: don't install symfony/property-access v7.0.6 (conflict analysis result)
- Root composer.json requires craftcms/cms 5.0.2 -> satisfiable by craftcms/cms[5.0.2].
- Root composer.json requires verbb/consume 2.0.0-beta.1 -> satisfiable by verbb/consume[2.0.0-beta.1].
```

### Steps to reproduce

1. Install Craft 5.0.2
2. Plugin Store -> Download `consume` using the `Try` button on DDEV.

### Craft CMS version

5.0.2

### Plugin version

2.0.0-beta.1

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/consume:"dev-craft-5 as 2.0.0-beta.2"`.

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.2](https://github.com/verbb/consume/releases/tag/2.0.0-beta.2)
